vertx-mysql-client/src/main/java/io/vertx/mysqlclient/MySQLConnectOptions.java

```diff
@@ -145,6 +146,26 @@ public MySQLConnectOptions setSsl(boolean ssl) {
     return this;
   }
 
+  /**
+   * Set the server RSA public key which is mostly used for encrypting password under insecure connections when performing authentication.
+   *
+   * @param serverRsaPublicKeyOptions the options of the server RSA public key
+   * @return a reference to this, so the API can be used fluently
+   */
+  public MySQLConnectOptions setServerRsaPublicKeyOptions(MySQLServerRsaPublicKeyOptions serverRsaPublicKeyOptions) {
```

Comment:
for the sake of simplicity I think we don't need a nested class but only `rsaPublicKeyPath` and `rsaPublicKeyValue` fields in these options

Reply:
I have changed this in https://github.com/eclipse-vertx/vertx-sql-client/pull/420/commits/2612e315b322a5c7335dfd56840bdcaa7f1ab824.